quil-rs/src/program/graph.rs

```diff
@@ -316,6 +317,7 @@ impl InstructionBlock {
 
         for (index, instruction) in instructions.iter().enumerate() {
             let node = graph.add_node(ScheduledGraphNode::InstructionIndex(index));
+            let instruction = *instruction;
```

Comment:
Was easier to dereference the `&&Instruction` here than to do `(*instruction)` multiple times below.